vision/product-search/cloud-client/src/main/java/com/example/vision/ProductInProductSetManagement.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
All the changes in this file can be removed, right?

Reply:
Yes, I did this intentionally.